tap-agent/src/agent/sender_account.rs

```diff
@@ -651,17 +656,16 @@ impl Actor for SenderAccount {
                     counter_greater_receipt_limit,
                     total_fee_greater_trigger_value,
                 ) {
-                    (true, _) => {
+                    (true, _) if !state.backoff_info.in_backoff() => {
```

Comment:
we don't need to check if it's calling directly an allocation, only if we'd call a rav for heaviest allocation